From 2e62b84638797f94daeb7d40964c34cce0527e7e Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Tue, 14 Aug 2018 16:51:24 +0300
Subject: [PATCH] btl: finish removal of UBOOT_SIZE

Now that the BTLID has taken the scratchpad 31, and since we do not
need to provide U-Boot size from BTL, finish the UBOOT_SIZE removal.

Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/board/habanalabs/common/btl/Makefile b/board/habanalabs/common/btl/Makefile
index 175981c..3dc91d4 100644
--- a/board/habanalabs/common/btl/Makefile
+++ b/board/habanalabs/common/btl/Makefile
@@ -32,13 +32,11 @@
 PRIMARY_CPP_FLAGS += -DSPL_OFFSET=0xa0000
 PRIMARY_CPP_FLAGS += -DSPL_SIZE=$(SPL_SIZE)
 PRIMARY_CPP_FLAGS += -DUBOOT_OFFSET=0xb0000
-PRIMARY_CPP_FLAGS += -DUBOOT_SIZE=$(UBOOT_SIZE)
 
 SECONDARY_CPP_FLAGS += $(COMMON_CPP_FLAGS)
 SECONDARY_CPP_FLAGS += -DSPL_OFFSET=0x8a0000
 SECONDARY_CPP_FLAGS += -DSPL_SIZE=$(SPL_SIZE)
 SECONDARY_CPP_FLAGS += -DUBOOT_OFFSET=0x8b0000
-SECONDARY_CPP_FLAGS += -DUBOOT_SIZE=$(UBOOT_SIZE)
 
 PRSTN_CPP_FLAGS += $(PRIMARY_CPP_FLAGS)
 PRSTN_CPP_FLAGS += -DPRSTN
diff --git a/board/habanalabs/common/post-image.sh b/board/habanalabs/common/post-image.sh
index 483ffef..8e4e7aa 100755
--- a/board/habanalabs/common/post-image.sh
+++ b/board/habanalabs/common/post-image.sh
@@ -47,7 +47,6 @@
 	dd status=none conv=sync bs=16 if=preboot.bin						 > ${HL_DEV}-preboot.bin
 	# prepare BTL image
 	BTL_CMD_ARGS="${BTL_CMD_ARGS} SPL_SIZE=$(printf "0x%X\n" `stat -c "%s" ${SPL_BINFILE}`)"
-	BTL_CMD_ARGS="${BTL_CMD_ARGS} UBOOT_SIZE=$(printf "0x%X\n" `stat -c "%s" ${UBOOT_BINFILE}`)"
 	cp -a ${BOARD_BASE}/../common/btl .
 	make -C btl clean all ${BTL_CMD_ARGS}
 	cp btl/sectors-be.bin .
